package.json

```diff
@@ -1,71 +1,71 @@
-{
-  "name": "react-scrolllock",
-  "description": "Prevent scroll on the body when component is mounted.",
-  "version": "3.0.0",
-  "main": "dist/index.js",
-  "license": "MIT",
-  "scripts": {
-    "test": "jest",
-    "start": "webpack-dev-server --mode development",
-    "build": "babel src -d dist --copy-files",
-    "prepublishOnly": "npm run build",
-    "build:examples": "webpack --mode production",
-    "deploy:examples": "gh-pages -d examples/dist",
-    "publish:examples": "npm run build:examples && npm run deploy:examples"
-  },
-  "jest": {
-    "setupFiles": [
-      "<rootDir>/config/setupPolyfills.js"
-    ],
-    "setupTestFrameworkScriptFile": "<rootDir>/config/setupFramework.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/jossmac/react-scrolllock.git"
-  },
-  "author": "Joss Mackison (https://github.com/jossmac)",
-  "bugs": {
-    "url": "https://github.com/jossmac/react-scrolllock/issues"
-  },
-  "homepage": "https://jossmac.github.io/react-scrolllock",
-  "keywords": [
-    "react",
-    "scroll",
-    "scroll-lock",
-    "disable-scroll"
-  ],
-  "dependencies": {
-    "exenv": "^1.2.2"
-  },
-  "peer-dependencies": {
-    "react": "^16.3.0"
-  },
-  "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.4",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-flow": "^6.23.0",
-    "babel-preset-react": "^6.24.1",
-    "css-loader": "^0.28.11",
-    "enzyme": "^3.2.0",
-    "enzyme-adapter-react-16": "^1.1.0",
-    "flow-bin": "^0.61.0",
-    "gh-pages": "^1.1.0",
-    "glam": "^5.0.1",
-    "html-webpack-plugin": "^3.2.0",
-    "jest": "^21.2.1",
-    "jest-cli": "^21.2.1",
-    "prop-types": "^15.6.0",
-    "raf": "^3.4.0",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
-    "react-prop-toggle": "^1.0.2",
-    "style-loader": "^0.21.0",
-    "webpack": "^4.5.0",
-    "webpack-cli": "^2.0.14",
-    "webpack-dev-server": "^3.1.3"
-  }
-}
+{

```

Comment:
ohgod did windows do this?

Reply:
probably prettier, it really messes with `package.json`